### ISSUE TITLE
🚨 SuperSync - Color Contrast Violations

### ISSUE THREAD

**Issue by OLLI-S** [super-productivity/super-productivity#5903]
### Version Used

https://sync.super-productivity.com/

### Expected Behavior

Regarding to the [Web Content Accessibility Guidelines (WCAG) 2.2](https://www.w3.org/TR/WCAG22/), the following criteria regarding the color contrast must be fulfilled:
* The Color Contrast Ratio of **textual elements** must be at least 4.5:1, for **large scale textual elements** (at least `18pt`/`24px` or bold and at least `14pt`/`18.5px`) the Color Contrast Ratio must be at least 3.0:1.
This applies to text and images of text.
See [Success Criterion 1.4.3 Contrast (Minimum)](https://www.w3.org/TR/WCAG22/#contrast-minimum)
* The Color Contrast Ratio of Non-Text elements must be at least 3.0:1, see [Success Criterion 1.4.11 Non-text Contrast](https://www.w3.org/TR/WCAG22/#non-text-contrast)
This refers to icons and to the visual presentation of UI components (e.g. the border lines of buttons, the checkmark of checkboxes, etc.

To fulfil the requirements of accessibility (A11Y) is also a law in Europe (European Accessibility Act, EN 301 549.

### Current Behavior

In the **SuperSync Account Page** some elements have not a sufficient contrast ratio:

<img width="570" height="731" alt="Image" src="https://github.com/user-attachments/assets/c0b71198-fdae-47d2-b47d-d5086f0bfe5f" />

<img width="583" height="839" alt="Image" src="https://github.com/user-attachments/assets/4c96d8ee-119f-40f6-b1da-2db280d7e48a" />

<img width="568" height="803" alt="Image" src="https://github.com/user-attachments/assets/7fbb2cb7-7145-4ecc-8bb6-7d4e8a0eb99a" />

<img width="579" height="1216" alt="Image" src="https://github.com/user-attachments/assets/02510d67-9b02-4856-8e7a-a616db793f7b" />

<img width="574" height="855" alt="Image" src="https://github.com/user-attachments/assets/5ff265b4-6d09-4483-9399-08130b7537d5" />

### Steps to Reproduce

Just read the comment https://github.com/super-productivity/super-productivity/issues/4988#issuecomment-3702478175 to get instructions, how to use the **Color Contrast Analyzer** (CCA)

### Can you reproduce this reliably?

Yes, I can reproduce it reliably.

### Console Output

```shell

```

**Comment by johannesjo** [comment]
Thank you very much for this! Will be fixed with the next release.
